src/helics/apps/CMakeLists.txt

```
@@ -99,7 +102,10 @@ if(BUILD_APP_LIBRARY)
     install(TARGETS helics_app DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT applications)
 
 	copy_key_files_to_target_location(helics_app)
-   
+
+	add_test(NAME app_version_return COMMAND helics_app --version)
+	add_test(NAME app_help_return COMMAND helics_app --help)
+    set_property(TEST app_version_return app_help_return PROPERTY LABELS Continuous)
   endif(BUILD_APP_EXECUTABLES)
```

I was thinking an additional test for the return value with help/version arguments to a subcommand, but from the way the helics_app handles subcommands, the return value should match help/version arguments to helics_app.

```suggestion

        add_test(NAME app_subcommand_version_return COMMAND helics_app player --version)
	add_test(NAME app_subcommand_help_return COMMAND helics_app player --help)
        set_property(TEST app_subcommand_version_return app_subcommand_help_return PROPERTY LABELS Continuous)
  endif(BUILD_APP_EXECUTABLES)
```